jupyterhub/values.yaml

```
@@ -22,6 +22,10 @@ imagePullSecrets: []
 # ConfigurableHTTPProxy speaks with the actual ConfigurableHTTPProxy server in
 # the proxy pod.
 hub:
+  config:
+    JupyterHub:
+      admin_access: true
+      authenticator_class: dummyauthenticator.DummyAuthenticator
```

shortcut works and is perhaps easier to understand:

```suggestion
      authenticator_class: dummy
```

We shouldn't be using dummyauthenticator anymore, as dummy ships with jupyterhub now.

---

Ah! So I'll remove jupyterhub-dummyauthenticator from the requirements.txt, and we deprecate the repsitory?

---

Yes, I believe so. This change is still needed to avoid the current crash loop backoffs because you've removed the `dummyauthenticator` package but are still using it in config.

---

Resolved by 5b0a011